Tests/XMTPTests/GroupTests.swift

```diff
@@ -83,11 +84,63 @@ class GroupTests: XCTestCase {
 		)
 	}
 
-	func testCanCreateGroups() async throws {
+	func testCanCreateAGroupWithDefaultPermissions() async throws {
 		let fixtures = try await localFixtures()
-		let group = try await fixtures.aliceClient.conversations.newGroup(with: [fixtures.bob.address])
+		let bobGroup = try await fixtures.bobClient.conversations.newGroup(with: [fixtures.alice.address])
+		try await fixtures.aliceClient.conversations.sync()
+		let aliceGroup = try await fixtures.aliceClient.conversations.groups().first!
+		XCTAssert(!bobGroup.id.isEmpty)
+		XCTAssert(!aliceGroup.id.isEmpty)
+		
+		
+		try await aliceGroup.addMembers(addresses: [fixtures.fred.address])
+		try await bobGroup.sync()
+		XCTAssertEqual(aliceGroup.memberAddresses.count, 3)
+		XCTAssertEqual(bobGroup.memberAddresses.count, 3)
+
+		try await aliceGroup.removeMembers(addresses: [fixtures.fred.address])
+		try await bobGroup.sync()
+		XCTAssertEqual(aliceGroup.memberAddresses.count, 2)
+		XCTAssertEqual(bobGroup.memberAddresses.count, 2)
 
-		XCTAssert(!group.id.isEmpty)
+		try await bobGroup.addMembers(addresses: [fixtures.fred.address])
+		try await aliceGroup.sync()
+		XCTAssertEqual(aliceGroup.memberAddresses.count, 3)
+		XCTAssertEqual(bobGroup.memberAddresses.count, 3)
+	}
+	
+	func testCanCreateAGroupWithAdminPermissions() async throws {
+		let fixtures = try await localFixtures()
+		let bobGroup = try await fixtures.bobClient.conversations.newGroup(with: [fixtures.alice.address], permissions: GroupPermissions.groupCreatorIsAdmin)
```

Comment:
XCode is giving me `Extra argument 'permissions' in call` @zombieobject am I doing this wrong?

Reply:
Must have just been an xcode issue. 🤔 The tests did pass.